CHANGES.rst

```diff
@@ -88,6 +88,8 @@ astropy.time
 astropy.timeseries
 ^^^^^^^^^^^^^^^^^^
 
+- A custom binning scheme is added in ``aggregate_downsample``. [#11266]
```

Comment:
This needs to be better clarified. Maybe specify what keyword you added and what is this custom binning good for?

Reply:
I'll add more details 🙂